### ISSUE TITLE
Unknown bug

### ISSUE THREAD

**Issue by ettern** [project-classic/ClassicCodex#8]
Seems to happen on quest changes, eg. objective complete, turn in quest.

```
Message: ...icCodex\lib\HereBeDragons\HereBeDragons-Pins-2.0.lua:689: HereBeDragons-Pins-2.0: AddWorldMapIconMap: 'uiMapID', 'x' and 'y' must be numbers
Time: Sun Aug 11 02:23:51 2019
Count: 1
Stack: ...icCodex\lib\HereBeDragons\HereBeDragons-Pins-2.0.lua:689: HereBeDragons-Pins-2.0: AddWorldMapIconMap: 'uiMapID', 'x' and 'y' must be numbers
[C]: in function `error'
...icCodex\lib\HereBeDragons\HereBeDragons-Pins-2.0.lua:689: in function `AddWorldMapIconMap'
Interface\AddOns\ClassicCodex\map.lua:529: in function `UpdateNodes'
Interface\AddOns\ClassicCodex\quest.lua:168: in function <Interface\AddOns\ClassicCodex\quest.lua:146>

Locals: (*temporary) = "HereBeDragons-Pins-2.0: AddWorldMapIconMap: 'uiMapID', 'x' and 'y' must be numbers"
```

**Comment by pgarneau** [comment]
What version are you using? Similar bug was fixed in version 1.0.1

**Comment by ettern** [comment]
Pretty sure I am using the latest version 1.0.0. The current .toc file is 1.0.0 atleast.
I've tried a clean install now, and will report back if it happens again.
